app/questionnaire/router.py

```diff
@@ -64,10 +69,14 @@ def get_last_location_in_questionnaire_url(self) -> str:
         routing_path = self.routing_path(*self._get_last_complete_section_key())
         return self.get_last_location_in_section(routing_path).url()
 
-    def is_list_item_in_list_store(self, list_item_id, list_name):
+    def is_list_item_in_list_store(
+        self, list_item_id: str, list_name: Optional[str]
+    ) -> bool:
         return list_item_id in self._list_store[list_name]
```

Comment:
If we didn't have a list name, this would error?

Reply:
This one is Optional because `list_name' is an optional attribute of Location class (which we use in here) so it's not always there potentially🤦‍♂️.

`list_name: Optional[str] = None`

Reply:
It just needs to be `str` for `list_name` and the before this is called, it needs to check you have a `list_name`.

Reply:
Added `str` type for `list_name` and extra conditional in parent method to check if `list_name` exists which lead to coverage error, so had to introduce extra test in `test_router.py`. Thanks!